test/conftest.py

```diff
@@ -14,7 +14,7 @@ def pytest_configure():
         DATABASES={
             "default": {
                 "ENGINE": "django_informixdb",
-                "SERVER": "informix",
+                "SERVER": "dev",
```

Comment:
This should remain `"informix"` so the tests can run against the informix container started by the docker-compose.yml file in this project.